.gitmodules

```
@@ -0,0 +1,6 @@
+[submodule "submodules/api-common-protos"]
+	path = submodules/api-common-protos
+	url = git@github.com:googleapis/api-common-protos.git
+[submodule "submodules/frequenz-api-common"]
+	path = submodules/frequenz-api-common
+	url = git@github.com:frequenz-floss/frequenz-api-common.git
```

Can we use `https` instead? It is less problematic (I had issues having a private key with a password that was locked and was suprised about being asked about my SSH key when pulling from a public repo):

```suggestion
	url = https://github.com/googleapis/api-common-protos.git
[submodule "submodules/frequenz-api-common"]
	path = submodules/frequenz-api-common
	url = https://github.com/frequenz-floss/frequenz-api-common.git
```